clients/baseclient.go

```diff
@@ -0,0 +1,15 @@
+package clients
+
+import (
+	"sync"
+	"github.com/ONSdigital/dp-rchttp"
+)
+
+// APIClient represents a common structure for any api client
+type APIClient struct {
+	BaseURL    string
+	AuthToken  string
```

Comment:
Potentially a separate issue, but shouldn't this not have AuthToken in it?

Reply:
no literally the thing I was trying to get rid of, just copy pasted without thinking - ty, good spot.